### PR TITLE
[Data Validation] Postgres AllDatatypes E2E test

### PR DESCRIPTION
**Pull request by aasthabharill**
- Fixed UUID handling during Hashing to add UUID support
- go/pg-bulk-migration-support-dd was followed as reference for mappings:

### CODE REVIEW

File: v2/gcs-spanner-dv/src/test/resources/BulkMigrationAndValidationPostgreSQLAllDataTypesE2EIT/postgresql-data-types.sql [hunk 1]
```diff
@@ -0,0 +1,1042 @@
+/*
+  NOTE: Boundary values injected in this schema strictly respect the target Spanner datatype limits.
```
Comment (darshan-sj) [inline v2/gcs-spanner-dv/src/test/resources/BulkMigrationAndValidationPostgreSQLAllDataTypesE2EIT/postgresql-data-types.sql:2]: Does this test run successfully? I think these comments would give errors when we try to apply to the database.

**Review by darshan-sj — APPROVED** [review]
Looks good

File: v2/gcs-spanner-dv/src/test/resources/BulkMigrationAndValidationPostgreSQLAllDataTypesE2EIT/postgresql-data-types.sql (hunk 1 above)
Comment (aasthabharill) [reply v2/gcs-spanner-dv/src/test/resources/BulkMigrationAndValidationPostgreSQLAllDataTypesE2EIT/postgresql-data-types.sql:2]: it does run successfully, because `/* .. */` is also a standard comment syntax in postgresql, its applied by the db engine and is ignored

Comment (aasthabharill) [reply v2/gcs-spanner-dv/src/test/resources/BulkMigrationAndValidationPostgreSQLAllDataTypesE2EIT/postgresql-data-types.sql:2]: test run passed as well
